dnf5/commands/repoquery/repoquery.cpp

```diff
@@ -272,6 +272,12 @@ void RepoqueryCommand::set_argument_parser() {
         *this, "recent", '\0', "Limit to only recently changed packages.", false);
     installonly = std::make_unique<libdnf::cli::session::BoolOption>(
         *this, "installonly", '\0', "Limit to installed installonly packages.", false);
+    srpm = std::make_unique<libdnf::cli::session::BoolOption>(
+        *this,
+        "srpm",
+        '\0',
+        "For selected packages format corresponding source RPMs (enables source repositories).",
```

Comment:
Hm, I don't understand the word "format" in this context. Previously, there was "operate on", which seems more intuitive, but maybe I am missing something?

Reply:
Well, other options are applied to the `srpms` - the query options. In dnf5, I don't see it in the documentation (at least not on the first glance), but there is `info` or the `pkg_attrs_options` (`requires`, `provides`...).

But I see what you mean and the "operate on" is not super clear either. The proposed sentence looks easier to understand.

Reply:
I have updated it.